mmdet3d/models/segmentors/base.py

```diff
@@ -16,6 +16,12 @@ class Base3DSegmentor(BaseSegmentor):
     data_dict and use a 3D seg specific visualization function.
     """
 
+    @property
+    def with_regularization_loss(self):
```

Comment:
with_correlation_loss

Reply:
Still, I think correlation loss may not be general enough?